### PR TITLE
Build and deploy script

### LINKED ISSUE #22: Deploy script
SSIA

### PR DESCRIPTION
**Pull request by syuhei176**
adding build script and deploy script to babylonnet.

Close #22

### CODE REVIEW

File: script/build.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+const spwanLigo = require('../test/helper/spawnLigo')
```
Comment (shogochiai) [inline script/build.js]: I found a typo.

File: script/deploy.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  const initialStorage = rmWhiteSpaces(
+    fs.readFileSync(path.join(buildDir, 'storage.tz')).toString()
+  )
+  try {
+    const deployCommand = getDeployCommand({
+      bytecode: parameter,
+      initialStorage: initialStorage
+    })
+    console.log('Deploying')
+    const outputOfDeploy = childProcess.execSync(deployCommand).toString()
```
Comment (shogochiai) [inline script/deploy.js]: When `npm run init-deploy` hasn't been running, 'Node is not running!' string is to be asigned to `outputOfDeploy` variable. This is being ignored currently and just showing "Deployed!" result.

File: script/init.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    __dirname,
+    'babylonnet.sh'
+  )} client -P 443 -A tezos-dev.cryptonomic-infra.tech -S config update`
+}
+
+function getClientActivate({ account }) {
+  return `${path.join(
+    __dirname,
+    'babylonnet.sh'
+  )} client activate account alice with '${JSON.stringify(account)}'`
```
Comment (shogochiai) [inline script/init.js]: I needed ` --force` to override existing alice account

Comment (shogochiai) [reply script/init.js]: I got an `Invalid activation. The public key tz1XuAz2HmWNMXKSV9GTx9zvo6w9Ngr8LWkW does not match any commitment.` error and this pubkey is from account.js which @syuhei176 was shared me.

I'm clarifying what is going on.

Comment (syuhei176) [reply script/init.js]: I think you don't need to `npm run init-deploy` if you already have account.

Comment (shogochiai) [reply script/init.js]: 1. When I forcibly update alice account, I could deploy and some michelson-level error has been shown to me.

2. After updating alice, cloudfrare returns "https://tezos-dev.cryptonomic-infra.tech:443/monitor/heads/main is timeout" error.

I'm just gonna clean up my accounts and try again.

Comment (shogochiai) [reply script/init.js]: Memo

I'm stuck in a ditch.
`babylonnet_node_1` and `babylonnet_upgrader_1` are keep restarting after running the `npm run init-deploy` command.
Maybe my computer has some misconfiguration due to have been dealt with the custom Tezos development.

Screen Shot 2020-01-09 at 13 15 56

File: script/build.js (hunk 1 above)
Comment (syuhei176) [reply script/build.js]: fixed

**Pull request merged**